tests/testLoadInputs.m

```diff
@@ -0,0 +1,61 @@
+classdef testLoadInputs < matlab.unittest.TestCase
+
+    properties
+        InputFilePathYAML
+        InputFilePathM
+    end
+
+    methods (TestMethodSetup)
+        function setUpPath(testCase)
+            % Get the full path to the current file
+            currentFilePath = mfilename('fullpath');
+            
+            % Get the directory of the current file
+            [currentDir, ~, ~] = fileparts(currentFilePath);
+            
+            % Add data folder to path
+            addpath([currentDir, '/data']);
+
+            % Construct the full path to the input file
+            testCase.InputFilePathM = fullfile(currentDir,  'data', 'inputFile.m');
```

Comment:
I changed the YAML input file defined in the folder 'inputFiles' and then ran the tests. 
I would have expected the tests to fail since my newly defined YAML file is not equal to the original .m file.
Shouldn't we add our YAML and .m files from the 'inputFiles' folder here?

Reply:
Maybe I misunderstood the purpose of the tests. Are the current tests written to check our loadInputs and validateInput functions and not the input file itself?

Reply:
The tests are indeed written for a different reason. They are to check that the input loading and data validation functionality works properly. They therefore use pre-defined input data files that are in the folder `tests/data`. If you change the functionality of the software, you would (likely) need to update the tests and possibly the test data. This is thus a tool for developers. 

Users should not have to rely on the tests to verify their inputs. Instead, they will rely on the data validation itself, which will be executed at runtime.

For your specific purpose (checking that the yaml loading results in the same output as the original `.m` file), we can add an assert statement to `run_awePower.m` After the appending the inputs, you can add

```matlab
% Sanity check yaml inputs
assert(isequal(inputs,inputFile_example_awePower()), 'Expected inputs to be identical to inputFile_example_awePower.m');
```
However, as you mentioned wanting to use only yaml input files, I imagine this would a temporary check.